encoding/openapi/build.go

```diff
@@ -858,11 +854,7 @@ func (b *builder) array(v cue.Value) {
 				core = b.core.items
 			}
 			t := b.schema(core, cue.AnyString, typ)
-			if len(items) > 0 {
```

Comment:
are you sure the logic here is correct?

the original one have a branching logic while your changes only handle 1